src/openassetio-python/cmodule/src/managerApi/ManagerInterfaceBinding.cpp

```diff
@@ -146,12 +146,12 @@ struct PyManagerInterface : ManagerInterface {
         RetainCommonPyArgs::forFn(successCallback), errorCallback);
   }
 
-  void preflight(const EntityReferences& entityReferences, const trait::TraitSet& traitSet,
+  void preflight(const EntityReferences& entityReferences, const trait::TraitsDatas& traitsDatas,
```

Comment:
[fix] Arg names need updating in the bindings below

Reply:
Fixed as part of d9e42e84 (then fixed the one you actually highlighted in 772c0670 :roll_eyes: )